AuroraEditor/Base/Editor/CodeEditorView/Model/Editor/Theme.swift

```diff
@@ -8,59 +8,10 @@
 
 import SwiftUI
 
-/// A code highlighting theme. Different syntactic elements are purely distinguished by colour.
-///
-public struct Theme: Identifiable {
-    public let id = UUID()
-
-    /// The name of the font to use.
-    public var fontName: String
-
-    /// The point size of the font to use.
-    ///
-    public var fontSize: CGFloat
-
-    /// The default foreground text colour.
-    public var textColour: OSColor
-
-    /// The background colour.
-    public var backgroundColour: OSColor
-
-    /// The colour of the current line highlight.
-    public var currentLineColour: OSColor
-
-    /// The colour to use for the selection highlight.
-    public var selectionColour: OSColor
-
-    /// The cursor colour.
-    public var cursorColour: OSColor
-
-    /// The colour to use if invisibles are drawn.
-    public var invisiblesColour: OSColor // NOTE: Currently unused
-
-    public init(fontName: String,
-                fontSize: CGFloat,
-                textColour: OSColor,
-                backgroundColour: OSColor,
-                currentLineColour: OSColor,
-                selectionColour: OSColor,
-                cursorColour: OSColor,
-                invisiblesColour: OSColor
-    ) {
-        self.fontName = fontName
-        self.fontSize = fontSize
-        self.textColour = textColour
-        self.backgroundColour = backgroundColour
-        self.currentLineColour = currentLineColour
-        self.selectionColour = selectionColour
-        self.cursorColour = cursorColour
-        self.invisiblesColour = invisiblesColour
-    }
-}
-
-/// A theme catalog indexing themes by name
-typealias Themes = [String: Theme]
+// This file is kept only for reference purposes
```

Comment:
```suggestion
// TODO: Remove this later
// This file is kept only for reference purposes
```

add the todo so we don't forget